src/components/20-molecules/datepicker/index.js

```diff
@@ -324,6 +336,39 @@ class AXADatepicker extends NoShadowDOM {
       });
     };
 
+    const versionedHtml = tagName => (strings, ...args) => {
+      const aVersion = AXADatepicker.version[1][tagName];
+      const oldTag = (closing = '') => `<${closing}${tagName}`;
+      const newTag = closing => `${oldTag(closing) + '-' + aVersion}`;
+      return html(
+        strings.map(string =>
+          string.replace(oldTag(), newTag()).replace(oldTag('/'), newTag('/'))
```

Comment:
new RegExp str.replace(/blue/g, "red");

Reply:
... Luca's comment is about multiple instances of the _same_ tagName in to-be-replaced string.

Additionally, a production version of a `versionedHTML` helper needs to generalize to multiple instances of _different_ tagNames, e.g. dropdown and button in same HTML template string.